BUILD

```diff
@@ -403,3 +403,31 @@ cc_test(
         "//external:gtest_main",
     ],
 )
+
+objc_library(
+    name = "protobuf_objc",
+    hdrs = glob([
+        "objectivec/*.h",
+        "objectivec/google/**/*.h",
+    ]),
+    includes = ["objectivec"],
+    non_arc_srcs = glob(
+        [
+            "objectivec/*.m",
+            # Timestamp.pbobjc.m and Duration.pbobjc.m are #imported by
+            # GPBWellKnownTypes.m. So we can't compile them separately
+            # (duplicate symbols).
```

Comment:
Actually, listing just the umbrella header and the umbrella implementation works alright, because Bazel doesn't move the sources around like Cocoapods.